src/lang/de.json

```diff
@@ -181,6 +181,8 @@
   "OSE.Setting.applyDamageOptionHint": "Auf welche(n) Token bei einem Angriffswurf Schaden angewendet werden soll",
   "OSE.Setting.damageSelected": "Ausgewählt",
   "OSE.Setting.damageTarget": "Ziel",
+  "OSE.Setting.InvertedCtrlBehavior": "Roll umkehren Ctrl/Meta-Taste",
+  "OSE.Setting.InvertedCtrlBehaviorHint": "Kehrt das Verhalten beim Halten von Ctrl/Meta beim Klicken auf eine Rolle um.",
```

Comment:
For future reference, please only commit English strings to source. We use https://crowdin.com/project/ose for translating strings

Reply:
Oooh I had no idea, thanks for letting me know, and: YAAAY :D